rocAL/rocAL_pybind/amd/rocal/readers.py

```diff
@@ -22,158 +22,184 @@
+                         "user_key_for_text": user_feature_key_map["image/class/text"], "user_key_for_xmin": user_feature_key_map["image/object/bbox/xmin"],
+                         "user_key_for_ymin": user_feature_key_map["image/object/bbox/ymin"], "user_key_for_xmax": user_feature_key_map["image/object/bbox/xmax"], 
+                         "user_key_for_ymax": user_feature_key_map["image/object/bbox/ymax"], "user_key_for_filename": user_feature_key_map["image/filename"]}
         for key in features.keys():
             if key not in user_feature_key_map.keys():
-                    print(
-                        "For Object Detection, ROCAL TFRecordReader needs all the following keys in the featureKeyMap:")
-                    print("image/encoded\nimage/class/label\nimage/class/text\nimage/object/bbox/xmin\nimage/object/bbox/ymin\nimage/object/bbox/xmax\nimage/object/bbox/ymax\n")
-                    exit()
-        tf_meta_data = b.tfReaderDetection(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+                print(
+                    "For Object Detection, ROCAL TFRecordReader needs all the following keys in the featureKeyMap:")
+                print("image/encoded\nimage/class/label\nimage/class/text\nimage/object/bbox/xmin\nimage/object/bbox/ymin\nimage/object/bbox/xmax\nimage/object/bbox/ymax\n")
+                exit()
+        tf_meta_data = b.tfReaderDetection(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     else:
         Pipeline._current_pipeline._reader = "TFRecordReaderClassification"
-        kwargs_pybind = {"path": path, "is_output": True, "user_key_for_label": user_feature_key_map[
-            "image/class/label"], "user_key_for_filename": user_feature_key_map["image/filename"]}
+        kwargs_pybind = {"path": path, "is_output": True, "user_key_for_label": user_feature_key_map["image/class/label"],
+                         "user_key_for_filename": user_feature_key_map["image/filename"]}
         for key in features.keys():
-                if key not in user_feature_key_map.keys():
-                    print(
-                        "For Image Classification, ROCAL TFRecordReader needs all the following keys in the featureKeyMap:")
-                    print("image/encoded\nimage/class/label\n")
-                    exit()
-        tf_meta_data = b.tfReader(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+            if key not in user_feature_key_map.keys():
+                print(
+                    "For Image Classification, ROCAL TFRecordReader needs all the following keys in the featureKeyMap:")
+                print("image/encoded\nimage/class/label\n")
+                exit()
+        tf_meta_data = b.tfReader(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     features["image/encoded"] = tf_meta_data
     features["image/class/label"] = labels
```

Comment:
Labels will be empty array?

Reply:
That value is not used anywhere so we just pass a empty list

Reply:
Should we pass if its not used?

Reply:
Even for the other readers, we pass empty lists to match the outputs from DALI readers